src/stac_iitmandi/templates/stac_iitmandi/_navbar.html

```diff
@@ -6,31 +6,31 @@
     </button>
     <div class="collapse navbar-collapse" id="main_nav">
         <ul class="navbar-nav ml-auto">
-            <li class="nav-item active">
+            <li class="nav-item {{home}}">
                 <a class="nav-link" href={% url 'stac-home' %}>Home <span class="sr-only">(current)</span></a>
             </li>
-            <li class="nav-item dropdown">
+            <li class="nav-item dropdown {{events_}}">
                 <a class="nav-link  dropdown-toggle" href="#" data-toggle="dropdown">Events</a>
                 <ul class="dropdown-menu">
-                    <li><a class="dropdown-item" href={% url 'astrax' %}>ASTRAX</a></li>
-                    <li><a class="dropdown-item" href={% url 'zenith' %}>ZENITH</a></li>
-                    <li><a class="dropdown-item" href={% url 'utkarsh' %}>UTKARSH</a></li>
+                    <li><a class="dropdown-item {{astrax}}" href={% url 'astrax' %}>ASTRAX</a></li>
+                    <li><a class="dropdown-item {{zenith}}" href={% url 'zenith' %}>ZENITH</a></li>
+                    <li><a class="dropdown-item {{utkarsh}}" href={% url 'utkarsh' %}>UTKARSH</a></li>
                 </ul>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#blogsite">Blog</a>
+                <a class="nav-link" href="#http://blog.pc.iitmandi.co.in/">Blog</a>
```

Comment:
Ideally this should be linked to [blog.stac.iitmandi.co.in](blog.stac.iitmandi.co.in). But for now you can leave it, we will add all links a the end of project.